### PR TITLE
fix: Remove non-settings version of example scripts

### PR DESCRIPTION
**Pull request by hpohekar**
closes https://github.com/ansys/pyfluent/issues/3014

1. Remove non-settings version of example scripts.
2. Replace TUI calls with settings wherever it is possible.

### CODE REVIEW

File: examples/00-fluent/exhaust_system_settings_api.py [hunk 1]
```diff
@@ -535,13 +535,6 @@
 )
 volume_mesh_gen.Execute()
 
-###############################################################################
-# Check mesh
-# ~~~~~~~~~~
-# Check the mesh.
-
-meshing.tui.mesh.check_mesh()
```
Comment (prmukherj) [inline examples/00-fluent/exhaust_system_settings_api.py:543]: If we are removing meshing.tui calls as well, then we shoul provide an alternative for mesh check, etc. Perhaps a scheme call?

Comment (seanpearsonuk) [reply examples/00-fluent/exhaust_system_settings_api.py:543]: @hpohekar  We have to retain TUI calls on the meshing side where there is no alternative. Scheme calls are an inferior alternative.

File: examples/00-fluent/external_compressible_flow.py [hunk 2]
```diff
@@ -199,19 +199,12 @@
...
-
-meshing.tui.mesh.check_mesh()
-
 ###############################################################################
 # Save mesh file
 # ~~~~~~~~~~~~~~
 # Save the mesh file (``wing.msh.h5``).
 
-meshing.tui.file.write_mesh("wing.msh.h5")
+meshing.meshing.File.WriteMesh(FileName="wing.msh.h5")
```
Comment (seanpearsonuk) [inline examples/00-fluent/external_compressible_flow.py:207]: :+1:

File: examples/00-fluent/modeling_cavitation.py [hunk 3]
```diff
@@ -306,7 +303,7 @@
 ###############################################################################
 # Mirror the display around the symmetry plane to show the full model.
 
-solver.tui.display.set.mirror_zones(["symm_2", "symm_1"])
+solver.results.graphics.views.mirror_zones = ["symm_2", "symm_1"]
```
Comment (seanpearsonuk) [inline examples/00-fluent/modeling_cavitation.py:306]: :+1: Caveat: we ought to start to use `solver.settings` (also below).

Comment (hpohekar) [reply examples/00-fluent/modeling_cavitation.py]: Done. Thank you.

File: examples/00-fluent/exhaust_system_settings_api.py (hunk 1 above)
Comment (hpohekar) [reply examples/00-fluent/exhaust_system_settings_api.py]: Done. Thank you.

**Review by prmukherj — APPROVED** [review]

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**